src/model/application.model.ts

```diff
@@ -17,7 +17,7 @@ export interface ApplicationData {
     entity?: entityType.Entity;
     beneficial_owners_statement?: beneficialOwnerStatementType.BeneficialOwnerStatementType;
     beneficial_owners_individual?: beneficialOwnerIndividualType.BeneficialOwnerIndividual[];
-    beneficialOwnerOther?: beneficialOwnerOtherType.BeneficialOwnerOther[];
+    beneficial_owners_other?: beneficialOwnerOtherType.BeneficialOwnerOther[];
```

Comment:
After this changes you need to modify as well the reference of `beneficialOwnerOther` to the following files:
- views/beneficial-owner-type.html
- views/includes/list/beneficial-owner.html 

And change the `corporationName` to match the new name field

Reply:
As well on line 28 of `src/model/application.model.ts`